hugegraph-server/hugegraph-api/src/main/java/org/apache/hugegraph/config/ServerOptions.java

```diff
@@ -321,4 +322,21 @@ public static synchronized ServerOptions instance() {
                     nonNegativeInt(),
                     1000L
             );
+
+    public static final ConfigOption<Double> JVM_MEMORY_MONITOR_THRESHOLD =
+            new ConfigOption<>(
+                    "memory_monitor.threshold",
+                    "Threshold for JVM memory usage monitoring, 1 means disabling the memory " +
+                    "monitoring thread.",
+                    rangeDouble(0.0, 1.0),
+                    0.85
+            );
+
+    public static final ConfigOption<Integer> JVM_MEMORY_MONITOR_PERIOD =
+            new ConfigOption<>(
+                    "memory_monitor.period",
+                    "The period of JVM memory usage monitoring, the unit is milliseconds.",
```

Comment:
prefer `The period in ms of JVM memory usage monitoring, in each period we will xx.`